src/services/users.service.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+const User = require('../models/user.model');
+
+const getAllUsers = async() => {
+  return User.findAll();
+};
+
+const getUsersById = async(id) => {
+  return User.findByPk(id);
+};
+
+const createUser = async(name) => {
+  const user = {
+    id: Math.floor(Math.random() * 10000),
```

Comment:
Let's add default value to the model itself, instead of generating some random number.

In the user model add:
```javascript
  id: {
      type: DataTypes.UUID,  // already exists
      defaultValue: DataTypes.UUIDV4,
      primaryKey: true // already exists
  }
  ```